swift/Tests/CoinAddressDerivationTests.swift

```diff
@@ -157,6 +157,11 @@ class CoinAddressDerivationTests: XCTestCase {
                 case .semux:
                     let expectedResult = "0xfe604170382452f77bc922bc19eb4b53504b09c2"
                     AssetCoinDerivation(coin, expectedResult, derivedAddress, address)
+                case .ark:
+                    let expectedResult = "Ac49m5pu5YpMMNgEbSYeZUEpRMHcSK3DfV"
+                    AssetCoinDerivation(coin, expectedResult, derivedAddress, address)
+                default:
```

Comment:
no need to specify `default`, it's done on purpose to make switch exhaustive

Reply:
True